dm/dm/master/openapi.go

```diff
@@ -614,6 +614,16 @@ func (s *Server) DMAPIGetTaskStatus(c *gin.Context, taskName string, params open
 				}
 			}
 		}
+		// add dump status
+		if dumpS := subTaskStatus.GetDump(); dumpS != nil {
```

Comment:
can you add a test in [ut](https://github.com/pingcap/tiflow/blob/f097a1294735d870b40e0738e54dd7f38cb0a4a3/dm/dm/master/openapi_test.go#L537) and [it](https://github.com/pingcap/tiflow/blob/f097a1294735d870b40e0738e54dd7f38cb0a4a3/dm/tests/openapi/run.sh#L266) for this new added fileds ?